includes/pages/guest_login.php

```diff
@@ -146,6 +146,28 @@ function guest_register() {
       $ok = false;
       $msg .= error(_("Please enter your Current City."), true);
     }
+    /**
+     * Google reCaptcha Server-Side Handling
+     */
+    if (isset($_REQUEST['g-recaptcha-response']) && !empty($_REQUEST['g-recaptcha-response'])) {
+      $curl = curl_init();
```

Comment:
Move reCaptcha calls to a new file `includes/sys_recaptcha.php` containing all needed functions to check with recaptcha. #142